### PR TITLE
Feature/user signin #160091310

### PR DESCRIPTION
**Pull request by nesh-dev**
### what does PR do ? 
Creates a signup form 
Creates a Signin form 

### Description of Task completed 
Create a html sigin page with a form 
create css file to style forms 

### How to test 

Clone this repo https://github.com/kevinene91/Fast-Food-Fast.git
navigate to the html sub folder inside the UI folder and click signup.html
navigate to the html sub folder inside the UI folder and click signin.html
- cd /Fast-Food-Fast/UI/html

### Relevant Pivotal Tracker story 
#160091310

ScreenShots

signup
login

### CODE REVIEW

**Comment by artorious** [comment]
Looks awesome. Great UI

**Comment by nesh-dev** [comment]
@artorious  Thansk appreciate your feedback

File: UI/assets/css/auth.css [hunk 1]
```diff
@@ -0,0 +1,8 @@
+body{
+    background-image: url("../images/fast_foods.jpg");
+    background-position: center center;
+    background-repeat: no-repeat;
+    background-size: cover;
+    background-attachment: fixed; 
+    font-family: Tahoma, sans-serif;
+}
```
Comment (JoyyToo) [inline UI/assets/css/auth.css:8]: add a new line at the end of this file and everywhere else you see the red marking

Comment (JoyyToo) [inline UI/assets/css/auth.css:8]: add a new line at the end of this file and everywhere else you see the red marking

**Pull request closed without merging**